docs/howtos/DataPlane Generation - DPG/05union.mdx

```diff
@@ -46,7 +46,31 @@ def completions(input: CompletionInput) -> CompletionResult:
 <TabItem value="csharp" label="CSharp" >
 
 ```csharp
-FIXME
+// Union "strict" | "lenient" will be generate as extensible enum
+public readonly partial struct Mode : IEquatable<Mode>
+{
+  public static Mode Strict { get; } = new Mode(StrictValue);
+  public static Mode Lenient { get; } = new Mode(LenientValue);
+}
+//protocol method
+public virtual async Task<Response> AnalyzeAsync(string mode, RequestContent content, RequestContext context = null)
+public virtual Response Analyze(string mode, RequestContent content, RequestContext context = null)
+//convenience method
+public virtual async Task<Response<AnalyzeResult>> AnalyzeAsync(Mode mode, BinaryData image, CancellationToken cancellationToken = default)
+public virtual Response<AnalyzeResult> Analyze(Mode mode, BinaryData image, CancellationToken cancellationToken = default)
```

Comment:
I think we should include the client class definition here as well otherwise this looks messy

Reply:
resolved @ArcturusZhang